k8s/elastic-gke-logging/schema.yaml

```diff
@@ -48,6 +48,16 @@ properties:
           - apiGroups: [""]
             resources: ["pods", "namespaces"]
             verbs: ["get", "list", "watch"]
+  metrics.image:
+    type: string
+    default: $REGISTRY/prometheus-to-sd:$TAG
+    x-google-marketplace:
+      type: IMAGE
+  metrics.enabled:
```

Comment:
Could we rename this variable to metrics.exporter.enabled? As I understand, with this configuration param we only decide if the exporter is enabled and metrics are always on.

Reply:
Discussed offline. Created an issue to track this - https://github.com/GoogleCloudPlatform/click-to-deploy/issues/474.